### PR TITLE
Feature#3/create models

### PR DESCRIPTION
**Pull request by claunicole**
# Description
-Create Models and migration of employees 
-Create Models and migration of company branch
-Add gems rubycritic, rubocop and sandi_meter

 # Testing
 -In the rails console, create a new employee with Employee.create and add a name, email, position, employee_num and private_number
-check if it is created correctly with Employee.all
-Do the same with the company branch

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -51,6 +51,9 @@ gem "bootsnap", require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'rubocop', require: false
+  gem "rubycritic", require: false
+  gem  'sandi_meter'
```
Comment (dannharo) [inline Gemfile]: check indentation and use single quotes for strings in ruby

File: app/models/employee.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Employee < ApplicationRecord
```
Comment (dannharo) [inline app/models/employee.rb:1]: is this model related to the `Company` Model? if yes, add the relation here 
your can read about that here:
https://guides.rubyonrails.org/association_basics.html

File: test/models/company_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CompanyTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (dannharo) [inline test/models/company_test.rb:4]: if you add the associations or validations, add some specs here please.

**Review by dannharo — COMMENTED** [review]
good job, just some comments ✌🏻

File: Gemfile (hunk 1 above)
Comment (JuanequeX) [reply Gemfile]: i agree with @dannharo

Comment (claunicole) [reply Gemfile]: Done

File: app/models/employee.rb (hunk 2 above)
Comment (claunicole) [reply app/models/employee.rb:1]: Done

File: test/models/company_test.rb (hunk 3 above)
Comment (claunicole) [reply test/models/company_test.rb:4]: Done

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by dannharo — APPROVED** [review]
`lgtm` :shipit:  :P
